mapping-for-navigation/_posts/0000-01-05-adding-exit-and-destination-signs.md

```diff
@@ -39,6 +39,12 @@ Destination tags describe the content of road signposts or ground information in
 - `destination:street=*` → is an alternative name for the numbered route – **only use in Florida, Kentucky, and Maryland**
 - `destination:lanes=*`, `destination:ref:lanes`, etc. → Often the destinations of a road differ from lane to lane. To specify those for every lane, `destination:lanes=*` is used.
 
+### Cardinal direction:
+Every dual carriage way has two oneways roads having traffic in opposite directions. Considering a dual carriage way as a destination, it is important to provide the cardinal direction of the way to which the route is navigating the user. To get this information we need to add cardinal directions to the `destination:ref` tags of the exit/entrance ways.
+
+<img width="632" alt="screen shot 2017-09-13 at 10 55 21 am" src="https://user-images.githubusercontent.com/8401827/30361068-97d03028-9872-11e7-9f89-f62439cb9b27.png"> 
```

Comment:
Can we check this image in? Otherwise, if this URL was generated by dragging an image into a text field for a comment that was never posted, GitHub may delete the image at any time.

Reply:
Most of our images are like this, lining up a cleanup of all images here: https://github.com/mapbox/mapping/issues/330